openapi/2023-12.yaml

```diff
@@ -473,4 +473,24 @@ components:
             sentAt:
               type: string
               description: UTC Date and time in ISO 8601 format
-              example: "2021-08-10T14:00:00.000+02:00"
\ No newline at end of file
+              example: "2021-08-10T14:00:00.000+02:00"
+    SMSUnsubscription:
+      type: object
+      required:
+        - type
+        - attributes
+      properties:
+        type:
+          type: string
+          example: "subscription"
```

Comment:
```suggestion
          example: "subscriptions"
```

Reply:
I think it should be single. We will send back data for 1 subscription.